mesheryctl/internal/cli/root/perf/result.go

```diff
@@ -221,8 +252,8 @@ func performanceResultsToStringArrays(results []models.PerformanceResult) ([][]s
 				P99:     P99,
 			},
 			StartTime:     result.TestStartTime,
-			MesheryID:     result.MesheryID,
-			LoadGenerator: result.RunnerResults.LoadGenerator,
+			MesheryID:     (*uuid.UUID)(mesheryid.Bytes()),
```

Comment:
This adds support for multi-cluster, is that right, @alphaX86, @Revolyssup, @hexxdump, @piyushsingariya?

Reply:
Yes, but if "None" provider is used, then a nil UUID is generated on both UserID and MesheryID which in turn causes panic error. That's why I made a way to bypass this so that instead of "" UUID, a blank UUID will be generated like 000-00000-0000-00000 (example)

But if it's not the case to be implemented then how can I bypass this? // @Revolyssup @hexxdump

Reply:
@alphaX86, I recognize that its typically not the case that I have a genuine question, but am usually insinuating an answer with my question... in this case, though, this is an honest question from me. :) You might be doing the right thing. I just didn't know if this was in part for `mesheryctl` to supporting querying different Meshery Servers (different  Meshery contexts / deployments).